plugins/node/opentelemetry-instrumentation-express/src/types.ts

```diff
@@ -80,6 +80,16 @@ export enum ExpressLayerType {
 
 export type IgnoreMatcher = string | RegExp | ((name: string) => boolean);
 
+export type SpanNameHook = (
+  request: Request,
+  route: string,
+  /** Type of the layer handling the request.
```

Comment:
Does tsdodc work inline in function arguments like this?

Reply:
Moved the explanation to README, got rid of these comments